packages/material-ui/src/Tooltip/Tooltip.js

```diff
@@ -386,14 +368,17 @@ const Tooltip = React.forwardRef(function Tooltip(props, ref) {
     }
   };
 
+  const handleMouseOver = handleEnter;
+  const handleMouseLeave = handleLeave;
```

Comment:
These can certainly be inlined but inlining them would make it easier to accidentally introduce different behavior. This approach makes it also easier to experiment with pointer events. We can spare the 10 bytes (`const a=a;`) for now. We're better served applying Closure compiler.